draft-wang-ppm-differential-privacy.md

```diff
@@ -236,10 +236,61 @@ the gap between actual number of Clients and minimum batch size.
 
 > TODO: Chris P to fill: user or report, given time
 
-## Privacy budget and accounting
+## Privacy budget and accounting {#budget}
 
-> TODO: Junye to fill, Pure ϵ-DP, (ϵ,δ)-approximate DP, (α,τ)-Renyi DP, Zero Concentrated-DP
-> TODO: talk about composability of DP
+There are various types of DP guarantees and budgets that can be enforced.
+Many applications need to query the Client data multiple times, for example:
+
+* Federated machine learning applications are usually composed of multiple
+  iterations.
```

Comment:
Can we reframe this in the DAP context? "Federated machine learning applications require multiple aggregates to be computed over the same underlying data."

Reply:
Rephrased it, and added "with different machine learning model parameters".